File: 1295 - Find Numbers with Even Number of Digits.c
int findNumbers(int* nums, int numsSize) {
    int result = 0;
    for(int i = 0; i < numsSize; i++){
        int number_digits = 0;
        while(nums[i]){
            nums[i] /= 10;
            number_digits++;
        }
        if(number_digits % 2 == 0)result++;
    }
    return result;
}

File: 1295 - Find Numbers with Even Number of Digits.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        result = 0
        for num in nums:
            number_digits = 0
            while num:
                num //= 10
                number_digits += 1
            if number_digits % 2 == 0:
                result += 1
        return result